enigmatry-entry-blueprint-app/src/app/core/auth/auth.interceptor.ts

```diff
@@ -25,6 +25,7 @@ export class AuthInterceptor implements HttpInterceptor {
       .pipe(
         switchMap(accessToken => next.handle(this.addAuthorizationHeader(req, accessToken))),
         catchError(async error => {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
           if (error instanceof HttpErrorResponse && error.status === HttpStatusCode.Unauthorized) {
```

Comment:
we are comparing status (number) === HttpStatusCode.Unauthorized (number)
this is not according to 'no-unsafe-enum-comparison'